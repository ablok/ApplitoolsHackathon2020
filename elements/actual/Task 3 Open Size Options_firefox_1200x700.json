[
  {
    ".custom-select-form li": [
      {
        "visibility": true,
        "location": {
          "x": 1007.6666870117188,
          "y": 700.2999877929688
        },
        "size": {
          "width": 176.3333282470703,
          "height": 32
        },
        "css": {
          "display": "list-item",
          "backgroundColor": "rgba(0, 77, 218, 0.04)",
          "backgroundImage": "none",
          "textDecoration": "rgb(68, 68, 68)",
          "color": "rgb(68, 68, 68)",
          "font": ""
        },
        "text": "Not Small"
      },
      {
        "visibility": true,
        "location": {
          "x": 1007.6666870117188,
          "y": 732.2999877929688
        },
        "size": {
          "width": 176.3333282470703,
          "height": 32
        },
        "css": {
          "display": "list-item",
          "backgroundColor": "rgba(0, 0, 0, 0)",
          "backgroundImage": "none",
          "textDecoration": "rgb(68, 68, 68)",
          "color": "rgb(68, 68, 68)",
          "font": ""
        },
        "text": "M"
      },
      {
        "visibility": true,
        "location": {
          "x": 1007.6666870117188,
          "y": 764.2999877929688
        },
        "size": {
          "width": 176.3333282470703,
          "height": 32
        },
        "css": {
          "display": "list-item",
          "backgroundColor": "rgba(0, 0, 0, 0)",
          "backgroundImage": "none",
          "textDecoration": "rgb(68, 68, 68)",
          "color": "rgb(68, 68, 68)",
          "font": ""
        },
        "text": "L"
      },
      {
        "visibility": true,
        "location": {
          "x": 1007.6666870117188,
          "y": 796.2999877929688
        },
        "size": {
          "width": 176.3333282470703,
          "height": 32
        },
        "css": {
          "display": "list-item",
          "backgroundColor": "rgba(0, 0, 0, 0)",
          "backgroundImage": "none",
          "textDecoration": "rgb(68, 68, 68)",
          "color": "rgb(68, 68, 68)",
          "font": ""
        },
        "text": "XL"
      }
    ]
  }
]
