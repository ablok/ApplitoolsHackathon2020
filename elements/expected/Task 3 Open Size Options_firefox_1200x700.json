[
  {
    ".custom-select-form li": [
      {
        "visibility": true,
        "location": {
          "x": 1007.6666870117188,
          "y": 699.2999877929688
        },
        "size": {
          "width": 176.3333282470703,
          "height": 32
        },
        "css": {
          "display": "list-item",
          "backgroundColor": "rgba(0, 77, 218, 0.04)",
          "backgroundImage": "none",
          "textDecoration": "rgb(68, 68, 68)",
          "color": "rgb(68, 68, 68)",
          "font": ""
        },
        "text": "Small (S)"
      },
      {
        "visibility": true,
        "location": {
          "x": 1007.6666870117188,
          "y": 731.2999877929688
        },
        "size": {
          "width": 176.3333282470703,
          "height": 32
        },
        "css": {
          "display": "list-item",
          "backgroundColor": "rgba(0, 0, 0, 0)",
          "backgroundImage": "none",
          "textDecoration": "rgb(68, 68, 68)",
          "color": "rgb(68, 68, 68)",
          "font": ""
        },
        "text": "M"
      },
      {
        "visibility": true,
        "location": {
          "x": 1007.6666870117188,
          "y": 763.2999877929688
        },
        "size": {
          "width": 176.3333282470703,
          "height": 32
        },
        "css": {
          "display": "list-item",
          "backgroundColor": "rgba(0, 0, 0, 0)",
          "backgroundImage": "none",
          "textDecoration": "rgb(68, 68, 68)",
          "color": "rgb(68, 68, 68)",
          "font": ""
        },
        "text": "L"
      },
      {
        "visibility": true,
        "location": {
          "x": 1007.6666870117188,
          "y": 795.2999877929688
        },
        "size": {
          "width": 176.3333282470703,
          "height": 32
        },
        "css": {
          "display": "list-item",
          "backgroundColor": "rgba(0, 0, 0, 0)",
          "backgroundImage": "none",
          "textDecoration": "rgb(68, 68, 68)",
          "color": "rgb(68, 68, 68)",
          "font": ""
        },
        "text": "XL"
      }
    ]
  }
]
